app/views/navigation_rules/edit.html.erb

```diff
@@ -26,51 +26,67 @@
 %>
 
 <%= render 'step_by_step_pages/nav', step_by_step_page: @step_by_step_page, active: "choose-navigation" %>
+<div class="govuk-grid-row">
+    <%= form_for(step_by_step_page_navigation_rules_path(@step_by_step_page), method: :put) do |form| %>
+      <div class="govuk-grid-column-full">
+        <table class="govuk-table table-stepnav-rules" data-module="filterable-table">
```

Comment:
If the table component is not permissive enough, we have a [table helper](https://github.com/alphagov/govuk_publishing_components/blob/d827e46d31cf091a90b8d1f2f774da06aa446c7c/lib/govuk_publishing_components/app_helpers/table_helper.rb) that may be helpful to preserve consistency of admin interfaces. It can be used like this: https://github.com/alphagov/content-publisher/blob/master/app/views/documents/index/_results.html.erb

Reply:
Thanks alex - I don't think we'll have these tables here for long, so once the redesign is done - I can have a look at your helper